src/middleware.ts

```diff
@@ -68,14 +69,18 @@ export const webhookHandler = factory.createHandlers(
       const adminUsernames: string[] = c.env.ADMIN_USERNAMES.split(',');
       const notionDatabaseId = c.env.NOTION_DATABASE_ID;
       const notionApiKey = c.env.NOTION_API_KEY;
+      const discordWH = c.env.DISCORD_WEBHOOK_URL;
```

Comment:
Hi @amanbairagi30, sir please add this environment variable in the `.example.dev.vars` This is IMPORTANT